README.md

```diff
@@ -88,11 +88,24 @@ This is very useful to inspect pipeline state during debugging, for example.
 - The state of the pipeline can easily be checked with the `@aside` macro
 - The `begin ... end` block marks very clearly where the macro is applied and works well with auto-indentation
 - Because everything is just lines with separate expressions and not one huge function call, IDEs can show exactly in which line errors happened
-- Pipe is a name defined by Base Julia which can lead to conflicts
+- Pipe (as used in [Pipe.jl](https://github.com/oxinabox/Pipe.jl)) is a name defined by Base Julia, which can lead to conflicts
 
-## Example
+## Usage
 
-An example with a DataFrame:
+A simple example that adds one and squares all elements of a range:
+
+```julia
+julia> @chain 1:3 begin
+         _ .+ 1
```

Comment:
In principle I'm not opposed to adding a simple non-DataFrame example. But `_ .+ 1` and `_ .^ 2` show the same syntax twice, which doesn't show off the range of options.

This for example shows a couple different things you can do:

```julia
@chain 1:10 begin
	  _ .^ 2
	  mod.(5)
	  filter(iseven, _)
	  sum
end
```

Reply:
What about this:

```julia
julia> names = ["ursula", "alice", "iris", "olga", "erica"]
5-element Vector{String}:
 "ursula"
 "alice"
 "iris"
 "olga"
 "erica"

julia> @chain names begin
    filter(contains("a"), _)
    @. uppercasefirst
    sort
    join(" and ")
    "$_ have an 'A' in their name."
end
"Alice and Erica and Olga and Ursula have an 'A' in their name."
```

Reply:
That's a good point — but I believe it makes sense for the very first example to aim for a demonstration of the basic structure of a chained block, rather than showcasing the variety of possibilities. With that in mind, I would maybe start with the first example you offered above, and show the second one next. This way we can keep the advantages of both. WDYT?